lib/spack/docs/example_files/test.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
I think that is the standard start of a YAML file (see http://yaml.org/), but all spack YAML files start with `<name>:`.  We should change this.